src/Utils/Terminal.java

```diff
@@ -0,0 +1,90 @@
+package Utils;
+
+import static Utils.ConsoleUtils.clearConsole;
+import static Utils.ConsoleUtils.print;
+import static Utils.FileUtils.analyzeBook;
+import static Utils.FileUtils.searchBook;
+import static Utils.FileUtils.writeStatisticsToFile;
+
+import Exceptions.CantReadFileException;
+import Exceptions.CantWriteToFileException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Terminal {
+
+  static Scanner scanner = new Scanner(System.in);
+  static Map<String, Integer> wordCounts = new HashMap<>();
+  static File bookFile;
+
+  public static void run() {
+    clearConsole();
+    print("Good day. Book reader terminal is loading...", true);
+    String input;
+
+    while (true) {
```

Comment:
while(true)  is a bad practice need to be avoided

Reply:
hmmm, do {} while() ?